c7n/filters/core.py

```diff
@@ -335,6 +335,11 @@ def process_value_type(self, sentinel, value):
             # comparisons is intuitively wrong.
             return value, sentinel
 
+        elif self.vtype == 'cidr_size':
+            for i in value:
+                v = parse_cidrs(i)
+            return sentinel, v
+
```

Comment:
In this context, The caller passes in the scalar and processes results against the list. And incidentally you can pass compare scalar to list or vice versa lots of extant examples.